cli/migrate/migrate.go

```diff
@@ -1850,7 +1851,12 @@ func (m *Migrate) ExportDataDump(tableNames []string) ([]byte, error) {
 	// to support tables starting with capital letters
 	modifiedTableNames := make([]string, len(tableNames))
 	for idx, val := range tableNames {
-		modifiedTableNames[idx] = fmt.Sprintf(`"%s"`, val) 
+		split := strings.Split(val, ".")
```

Comment:
I think we should be throwing an error if `len(split)` isn't either 2 or 1, as it's not a valid table/schema name.

Reply:
Sounds good! thank you.

Reply:
@kolharsam let me know if this is ready to go